marketplace/prometheus.json

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
Marketplace has been deprecated. Could you help us updating the corresponding specification for Prebuilt?